src/main/java/uk/gov/hmcts/reform/ethos/ecm/consumer/config/ServiceBusUpdateCaseReceiverConf.java

```diff
@@ -15,8 +15,13 @@
 
 @AutoConfigureAfter(ServiceBusCreateUpdatesReceiverConf.class)
 @Configuration
+@SuppressWarnings("PMD")
 public class ServiceBusUpdateCaseReceiverConf {
 
+    private final IQueueClient updateCaseListenClient;
```

Comment:
You should also move updateCaseListenExecutor and messageHandlerOptions

Reply:
moved